ansible/roles/ocp4-workload-integreatly/defaults/main.yml

```diff
@@ -24,4 +24,27 @@ silent: False
 ocp4_workload_integreatly_defaults:
     variable_1: "value 1"
     variable_2: "value 2"
-    variable_secret: ""
\ No newline at end of file
+    variable_secret: ""
+
+action: install
+namespace: redhat-rhmi-3scale
+
+seed_secret_name: system-seed
+
+tmp_dir: /tmp
+
+# default tenant details
+tenant_name: admin-tenant
+tenant_admin_user: admin
+tenant_admin_email: admin@example.com
+tenant_admin_password: Password1
+tenant_detail_secret_name: "{{ tenant_name }}"
+tenant_detail_secret_namespace: "{{ namespace }}"
+tenant_id_format: evals%02d-tenant
+
+lab_users_name_format: evals%02d
+lab_user_count : 50
```

Comment:
We probably could remove `lab_` prefix from these variables

Reply:
:+1: